### PR TITLE
Fix speech recognition demo

### PR DESCRIPTION
**Pull request by AlexeyKruglov**
Replace contexttimer module; fix the broken alphabet file example

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/python_demos/speech_recognition_demo/utils/context_timer.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import time
+
+
+class Timer:
+    def __enter__(self):
+        self.elapsed = None
+        self.start = time.time()
```
Comment (IRDonch) [inline demos/python_demos/speech_recognition_demo/utils/context_timer.py:7]: You should be using `time.perf_counter()` instead.

File: demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf [hunk 2]
```diff
@@ -1,8 +1,7 @@
-# Each line in this file represents the Unicode codepoint (UTF-8 encoded)
-# associated with a numeric label.
-# A line that starts with # is a comment. You can escape it with \# or \\ if you wish
-# to use '#' or '\' as a label.
-
+# Each line in this file represents the Unicode codepoint (UTF-8 encoded) associated with a numeric label.
+# A line that starts with # is a comment. You can escape it with \# or \\ if you wish to use '#' or '\' as a label.
+# The following line in this example contains a single whitespace character, which is the first symbol of the alphabet:
+
```
Comment (IRDonch) [inline demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf:4]: I'm not a fan of formats that require invisible whitespace. Maybe you could change the demo to use `shlex.split` to parse this file? That would be compatible with the current syntax, and you could express this character as `' '`.

**Review by IRDonch — CHANGES_REQUESTED** [review]

File: demos/python_demos/speech_recognition_demo/utils/context_timer.py (hunk 1 above)
Comment (generalova-kate) [reply demos/python_demos/speech_recognition_demo/utils/context_timer.py:7]: Please use timeit.default_timer() it is a canonical way

Comment (AlexeyKruglov) [reply demos/python_demos/speech_recognition_demo/utils/context_timer.py]: Done, with timeit..

File: demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf (hunk 2 above)
Comment (AlexeyKruglov) [reply demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf]: I'd like to keep backward compatibility with Mozilla's ctcdecode alphabet format. So I changed to accept both whitespace and \w, and put \w into the example file.

File: demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf [hunk 3]
```diff
@@ -1,8 +1,7 @@
-# Each line in this file represents the Unicode codepoint (UTF-8 encoded)
-# associated with a numeric label.
-# A line that starts with # is a comment. You can escape it with \# or \\ if you wish
-# to use '#' or '\' as a label.
-
+# Each line in this file represents the Unicode string (UTF-8 encoded) associated with the position in the alphabet.
+# A line that starts with "#" is a comment. Escape the first character with "\" to use '#' or '\' as a label.
+# Whitespace sybmol can be entered as a whitespace character on a separate line or with "\w".
+\w
```
Comment (IRDonch) [inline demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf:4]:
```suggestion
# A space character can be entered as "\s".
\s
```

"Whitespace" is a general term that includes spaces, tabs and linebreaks. Here you're specifically talking about spaces.

Comment (AlexeyKruglov) [reply demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf]: Let's split then this PR into two PRs: I'll keep the timer fix in this PR, and will make a new PR with example alphabet file fix.  I removed this commit from this PR (check "Files changed" tab).

Comment (IRDonch) [reply demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf]: Don't know why it was needed, but sure, works for me.

**Review by IRDonch — APPROVED** [review]

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**